app/forms/claim_submission_base_form.rb

```diff
@@ -35,20 +34,30 @@ def save
 
   delegate :answers, to: :journey_session
 
-  def build_claim
-    claim = Claim.new
+  def main_eligibility
+    @main_eligibility ||= eligibilities.first
+  end
```

Comment:
All the subclasses had this method, so I moved it here.
The only subclass with a different implementation is `Journeys::AdditionalPaymentsForTeachingClaimSubmissionForm`